src/components/wizard/WizardStep.js

```diff
@@ -0,0 +1,159 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {StyleSheet} from 'react-native';
+import View from '../view';
+import Text from '../text';
+import Image from '../image';
+import TouchableOpacity from '../touchableOpacity';
+import {PureBaseComponent} from '../../commons';
+import Colors from '../../style/colors';
+import BorderRadiuses from '../../style/borderRadiuses';
+import Spacings from '../../style/spacings';
+import {States, StatesConfig} from './WizardStates';
```

Comment:
States is not being used, it will fail lint

Reply:
Added state to props, it is now being used